src/landing/LandingPage.js

```diff
@@ -1,14 +1,91 @@
 import React, { useState } from 'react';
+import { gql, useLazyQuery, useMutation } from '@apollo/client';
 import {
   Paper, Tab, Tabs,
 } from '@material-ui/core';
 import LoginForm from './LoginForm';
 import './styles.css';
 import RegisterForm from './RegisterForm';
 
+const GET_USERS = gql`
+  query Login{
+    users{
+      username
+      password
+    }
+  } 
+`;
+
+const CREATE_USER = gql`
+  mutation Register($username: String!, $firstName: String!, $lastName: String!, $password: String!){
+    createUser(username: $username, 
+      firstName: $firstName, 
+      lastName: $lastName, 
+      password: $password,
+      userRoleId: 3){
+         username,
+      }
+}
+`;
+
 const LandingPage = () => {
   const [form, setForm] = useState('Login');
 
+  const [getUsers, {
+    data: queryData, error: queryError,
+  }] = useLazyQuery(GET_USERS, {
+    notifyOnNetworkStatusChange: true,
+    fetchPolicy: 'cache-and-network',
+  });
+
+  const submitLogin = (username, password) => {
+    getUsers();
+
+    if (queryError) {
+      console.log(queryError);
+      alert('Something went wrong');
+      return;
+    }
+
+    let loginStatus = false;
+    if (queryData) {
+      queryData.users.forEach((user) => {
```

Comment:
You could try to handle this with the help of Array.find()

Reply:
updated (modified structure)